portfolio/src/main/webapp/index.html

```diff
@@ -45,6 +45,21 @@ <h1 class="about-text">About</h1>
         Women's Ultimate Frisbee Team and am a TA for CIS110: Introduction to Programming.   
       </p>
     </div> 
+    
+    <div class="project">
+      <h1 id="project">Servlet Tests</h1>
+      <div class="divider"></div>
+      <button onclick="getJson()">Fetch Comments</button>
```

Comment:
`getJson()` is a bit ambiguous can we name this something like `fetchComments()`?